packages/react-dom/src/__tests__/ReactCompositeComponent-test.js

```diff
@@ -428,7 +428,8 @@ describe('ReactCompositeComponent', () => {
       'Cannot update during an existing state transition (such as within ' +
         "`render` or another component's constructor). Render methods should " +
         'be a pure function of props and state; constructor side-effects are ' +
-        'an anti-pattern, but can be moved to `componentWillMount`.',
+        'an anti-pattern, but can be moved to `getDerivedStateFromProps`.\n' +
```

Comment:
get rid of `componentWillMount`

Reply:
Side effects *cannot* be moved to gDSFP, it has to be pure. Let’s revert this part for now? We can think about this message separately